.vscode/settings.json

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
I'm wondering if single vscode settings for both packages inside this repo works. Generally I put them inside each package so that they can point to a different catkin_virtualenv and whatnot.  Prevents you from being forced to use one package to develop the other.